content/01.abstract.md

```diff
@@ -11,3 +11,21 @@ Publication with Manubot is continuous:
 When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
 Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers (e.g. DOIs, PubMed IDs, ISBNs, URLs), whose metadata is then retrieved and converted to a user-specified style.
 Manubot modernizes publishing to align with the ideals of open science by making it transparent, reproducible, immediate, versioned, collaborative, and free of charge.
+
+## Author summary
+
+Traditionally, scholarly manuscripts, such as this one you are reading, have been written in private.
+This worked because the research was also performed in private by a predefined team of collaborators.
+But now the internet has enabled realtime open science, where project communication occurs online in a public venue and anyone is able to contribute.
+But how can dispersed teams of online contributors jointly prepare manuscripts to describe their work?
+
+Existing tools fail to scale and struggle to enable iterative refinement of proposed changes.
+Therefore, we created a system for writing manuscripts on GitHub, called Manubot.
```

Comment:
We can explain GitHub to make it more accessible: `on the version control platform GitHub`.  That exact phrase may not be sufficiently accessible though.

Reply:
A bit less jargon:

> Therefore, we created a system for writing collaborative manuscripts incorporating automatic versioning, called Manubot.

Reply:
I agree we don't need to name GitHub here.  It appears in the abstract.